BoxBrowseSDKSampleApp/Podfile.lock

```diff
@@ -15,11 +15,11 @@ DEPENDENCIES:
 
 EXTERNAL SOURCES:
   box-ios-browse-sdk:
-    :path: "../"
```

Comment:
We actually want the quotation marks, the removal is because you're on an older version of Ruby.